move/axelar_gateway/sources/gateway.move

```diff
@@ -126,6 +116,7 @@ fun init(ctx: &mut TxContext) {
 }
 
 /// Setup the module by creating a new Gateway object.
+#[allow(lint(share_owned))]
```

Comment:
what's this for? is there a workaround to avoid it? If not, explain the reason in a comment

Reply:
removed